tasks/setup.yaml

```
@@ -1,6 +1,39 @@
+variables:
+  - name: REGISTRY1_TEST_IMAGE
+    default: "registry1.dso.mil/ironbank/opensource/defenseunicorns/zarf/zarf-agent"
+
 tasks:
   - name: k3d-test-cluster
     actions:
       - description: Create k3d cluster with UDS Core Istio
         # renovate: datasource=github-tags depName=defenseunicorns/uds-core versioning=semver
         cmd: uds deploy oci://defenseunicorns/uds/bundles/k3d-core-istio-dev:0.10.0-${UDS_ARCH} --confirm --no-progress
+
+  - name: registry-login
+    inputs:
+      registry:
+        default: ${REGISTRY}
+        description: "Registry"
+      registryUsername:
+        default: ${REGISTRY_USERNAME}
+        description: "Registry username"
+      registryPassword:
+        default: ${REGISTRY_PASSWORD}
+        description: "Registry password"
+      registryRetryInterval:
+        default: ${REGISTRY_RETRY_INTERVAL}
```

```suggestion
        description: "Registry"
      registryUsername:
        description: "Registry username"
      registryPassword:
        description: "Registry password"
      registryRetryInterval:
        default: "5"
```

So my leaning is to do this - drop the `default: ${...}` stuff and add a single default for the retry interval. We don't need these defaults if we have the assumption that this task will be always be imported (which I think is a safe assumption given that these are common tasks designed to be imported). Definitely call out any scenarios this wouldn't work with - I played with these tasks a bit but I know there was more stress testing by others.